<?xml version="1.0"?>
<launch>


<node pkg="auto_complete_graph" type="acg_node_localization" name="acg_node_localization" output="screen">

	<param name="use_prior" value="true" />
	<param name="use_robot_maps" value="true" />
	<param name="use_corner" value="true" />
	<param name="corner_covariance" value="false" /> <!-- DOESNT WORK -->
	<param name="own_registration" value="true" />
	<param name="link_to_prior" value="false" />
	<param name="mcl_observation_on_prior" value="true" />
	<param name="links_prior_classic_ssrr" value="false" />
	<param name="world_frame" value="/world" />
	<param name="sensor_frame" value="/velodyne" />
	<param name="covariance_to_find_links" value="false" />
	<param name="gaussian_scaling_factor" value="2" />
	<param name="threshold_score_link_creation" value="0.5" />

    <remap from="acg_node_localization/prior_ndt" to ="/ndt_map_init_mcl"/>
 <!-- /auto_complete_graph_rviz_small_optimi -->

  </node>


</launch>
